styles/components/modal.css

```diff
@@ -0,0 +1,38 @@
+.modal-cont {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
```

Comment:
height by default is 100%, and try to define top 0 and left 0 in HTML tag so it will be inherited by default.

Reply:
This is for a particular modal component. that's why I have explicitly given the height as 100%.